test/compress.js

```diff
@@ -147,6 +147,7 @@ function run_compress_tests() {
                 var input = as_toplevel(test.input, test.mangle);
                 var input_code = make_code(input, output_options);
                 var input_formatted = make_code(test.input, {
+                    ecma: 2015,
```

Comment:
This was to support Surrogates in tests when erroring with formatted input code.